projections/shapes/lib/shapes/client.rb

```diff
@@ -57,11 +55,8 @@ class Client < Smithy::Client::Base
     #  When true, the request will sleep until there is sufficient client side capacity to retry
     #  the request. When false, the request will raise a `CapacityNotAvailableError` and will
     #  not retry instead of sleeping.
-    # @option options [ShapeService::AuthResolver] :auth_resolver
-    #  The auth resolver used to resolve authentication. Any object that responds to `#resolve(parameters)`.
-    # @option options [Hash] :auth_schemes
-    #  The auth schemes used to resolve authentication. The key is the scheme name as a String,
-    #  and the value is an initialized auth scheme class.
+    # @option options [#resolve(context)] :auth_resolver (<DEFAULT_AUTH_RESOLVER>)
```

Comment:
DEFAULT_AUTH_RESOLVER needs to be populated with a real value here. Check how this is done for DEFAULT_PROTOCOL, but we do a gsub.